examples/README.md

```diff
@@ -0,0 +1,31 @@
+# wolfEngine Examples
+
+This directory contains example programs using wolfEngine on Linux. See below for more on each. Each program is also documented in its source code.
+
+## Setting up wolfEngine
+
+See README.md in the root source code directory for instructions on building wolfEngine. Configure wolfEngine with `--enable-debug` (or `-DWOLFENGINE_DEBUG` if not using the configure script) to see debug output from wolfEngine during execution.
+
+## Building
+
+To compile an example, run
+
+```
+gcc -I/path/to/openssl/include -L/path/to/openssl/lib example.c -o example -lcrypto -g3 -O0
```

Comment:
Prefer '<example>'.
Makes clear that you replace with something.
Alternatively, put exact instructions for each example.